@navikt/core/react/src/chips/chips.stories.tsx

```diff
@@ -230,52 +184,24 @@ export const Small = () => {
 };
 
 export const ColorRole = () => (
-  <div>
+  <VStack gap="space-24">
     <div>
-      <Regular />
+      <Medium />
     </div>
     <div data-color="brand-magenta">
-      <Regular />
+      <Medium />
     </div>
     <div>
-      <Regular data-color="brand-magenta" />
+      <Medium data-color="brand-beige" />
     </div>
-  </div>
+  </VStack>
 );
 
-export const Chromatic: Story = {
-  render: () => (
-    <div>
-      <div>
-        <h2>Default</h2>
-        <Default />
-      </div>
-      <div>
-        <h2>Toggle</h2>
-        <h3>Medium</h3>
-        <Toggle size="medium" />
-        <h3>Small</h3>
-        <Toggle size="small" />
-      </div>
-      <div>
-        <h2>Removable</h2>
-        <Removable />
-      </div>
-      <div>
-        <h2>Regular</h2>
-        <Regular />
-      </div>
-      <div>
-        <h2>Small</h2>
-        <Small />
-      </div>
-      <div>
-        <h2>ColorRole</h2>
-        <ColorRole />
-      </div>
-    </div>
-  ),
-  parameters: {
-    chromatic: { disable: false },
-  },
-};
+export const Chromatic = renderStoriesForChromatic({
+  Toggle,
+  Removable,
+  Medium,
```

Comment:
Looks like it's redundant to have both Medium and Toggle+Removable stories in Chromatic.

```suggestion
  Medium,
```


```suggestion
  Toggle,
  Removable,
```